djangomako/backends.py

```diff
@@ -164,7 +165,37 @@ def render(self, context=None, request=None):
             context['csrf_input'] = csrf_input_lazy(request)
             context['csrf_token'] = csrf_token_lazy(request)
 
-        return self.template.render(**context)
+        try:
+            return self.template.render(**context)
+        except Exception as e:
+            tb = RichTraceback()
+
+            lines = tb.source
```

Comment:
Rename `lines` to `template_source`

Reply:
sure `lines` is should be renamed, but like `traceback` object argument name is `source` then imho `lines` should be rename `source`.

Let me know your through